src/TraceParserGen.Tests/TestBase.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.IO;
+using Xunit.Abstractions;
+
+namespace TraceParserGen.Tests
+{
+    /// <summary>
+    /// Base class for TraceParserGen tests
+    /// </summary>
+    public abstract class TestBase
+    {
+        protected static string OriginalInputDir = FindInputDir();
+        protected static string TestDataDir = Path.GetFullPath("inputs");
+        protected static string BaseOutputDir = Path.GetFullPath("output");
+
+        protected TestBase(ITestOutputHelper output)
+        {
+            Output = output;
+            OutputDir = Path.Combine(BaseOutputDir, Guid.NewGuid().ToString("N").Substring(0, 8));
+            
+            // Ensure output directory exists
+            if (!Directory.Exists(OutputDir))
+            {
+                Directory.CreateDirectory(OutputDir);
+            }
+        }
+
+        protected ITestOutputHelper Output { get; }
+
+        protected string OutputDir { get; }
+
+        /// <summary>
+        /// Finds the input directory for test files
+        /// </summary>
+        private static string FindInputDir()
+        {
+            string dir = Environment.CurrentDirectory;
+            while (dir != null)
+            {
+                string candidate = Path.Combine(dir, @"TraceParserGen.Tests\inputs");
```

Comment:
I think this should just be "inputs" because the inputs are being copied into the output directory and the output directory will be the current directory when running.